src/components/specific/tags/tags-main/TagsMain.vue

```diff
@@ -0,0 +1,209 @@
+<template>
+  <div ref="tagsMain" class="tags-main">
+    <div class="tags-main__content">
+      <div class="tags-main__content__header">
+        <div class="tags-main__content__header__left-side">
+          <BIMDataIcon
+            name="tag"
+            fill
+            color="primary"
+            size="s"
+            margin="2.5px 0 0 0"
+          />
+          <span>{{ $t("Tag.title") }}</span>
+        </div>
+        <div class="tags-main__content__header__right-side">
+          <BIMDataButton ghost rounded icon @click="$emit('close')">
+            <BIMDataIcon name="close" size="xxs" fill color="granite-light" />
+          </BIMDataButton>
+        </div>
+      </div>
+      <div class="tags-main__content__list-add">
+        <span>{{ $t("Tag.list") }}</span>
+        <BIMDataButton
+          color="primary"
+          radius
+          ghost
+          icon
+          margin="12px"
+          @click="showAddTagInput = true"
+        >
+          <BIMDataIcon
+            name="close"
+            size="xxxs"
+            margin="0 calc(var(--spacing-unit) / 2) 0 var(--spacing-unit)"
+            :rotate="45"
```

Comment:
why `:` is the value is constant?

Reply:
The rotate prop is expecting a `Number` value, so we have to bind it this way, otherwise it will be a string by default.